normalize.js

```diff
@@ -111,11 +126,8 @@ define(function() {
     
     return out.substr(0, out.length - 1);
   };
-  
-  var normalizeCSS = function(source, fromBase, toBase) {
 
-    fromBase = removeDoubleSlashes(fromBase);
-    toBase = removeDoubleSlashes(toBase);
+  var normalizeCSS = function(source, fromBase, toBase) {
```

Comment:
what do you mean?

It is on second indent level (inside the first-level `define(function(){`) so would be two spaces, and there are two spaces before `var normalizeCSS`.